modules/network_load_balancer/main.tf

```
@@ -0,0 +1,79 @@
+resource "aws_lb" "tfe_lb" {
+  name               = "${var.friendly_name_prefix}-tfe-web-alb"
```

A few instances of "alb" in this file that should be changed to "nlb".

```suggestion
  name               = "${var.friendly_name_prefix}-tfe-nlb"
```